### ISSUE TITLE
Getting started example does not work

### ISSUE THREAD

**Issue by pradel** [phenomic/phenomic#1251]
Hi!
I first followed the getting started section on the website and it looks like the list of posts is always empty (the `posts` prop).
I then trying to clone and run the getting started example and same, the list is empty on the example.
Am I doing something wrong or the example is broken?
Thanks :)

**Comment by MoOx** [comment]
Yeah really sorry about that. I need to cut a new release as docs reflect a minor breaking change in master. See https://gitter.im/phenomic/phenomic?at=5ae9c69cf2d2d53704593c22

**Comment by MoOx** [comment]
I will leave it open until I cut a release, which you can expect in the coming hours/days depending on the mess in my life :)

**Comment by pradel** [comment]
Ahah, got it working with the branch docs thanks :)
